__tests__/app.test.js

```diff
@@ -71,5 +71,42 @@ describe('GET article by id', () => {
 
 
         })
+        describe('PATCH inc_votes by ID', () => {
+            describe('PATCH inc_votes by ID', () => {
+                test('returns status 404 when id not found ', () => {
+                    return request(app)
+                    .patch('/api/articles/99')
+                    .expect(404)
+                   })
+                })
+                test('returns status 400 when given invalid id ', () => {
+                    return request(app)
+                    .patch('/api/articles/test')
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('Bad Request');
+                    })
+                })
+                    test('returns status 400 when given invalid vote increment ', () => {
+                        return request(app)
+                        .patch('/api/articles/1')
+                        .send({inc_votes : test})
```

Comment:
test should be in a string to avoid a false fail from an undefined 'test' variable